From dd2ce5ff2d18b5a89ad6558133c0552ed47e8bc1 Mon Sep 17 00:00:00 2001
From: allen yan <yanwei@marvell.com>
Date: Wed, 26 Apr 2017 10:35:48 +0800
Subject: [PATCH 0100/1239] arm64: dts: armada-3720-db: add dts for more
 devel-board flavors

This patch added dts support for both rev1.x, rev2.x and rev 3.x ECO;
This patch added dts support for boards of different ddr types.
Suffix "A" dts configurates comphy0 with usb3_host;
Suffix "B" dts configurates comphy0 with sgmii1;
Suffix "C" dts configurates comphy0 with sgmii1-sfp2.5g.

Change-Id: Ie57dd8ff9c3066efcbed7a0fc9651f7fd9a6400b
Signed-off-by: allen yan <yanwei@marvell.com>
---
 arch/arm/dts/Makefile                     |  9 ++++
 arch/arm/dts/armada-3720-ddr3-db-v1-A.dts | 43 ++++++++++++++++
 arch/arm/dts/armada-3720-ddr3-db-v1-B.dts | 55 ++++++++++++++++++++
 arch/arm/dts/armada-3720-ddr3-db-v1-C.dts | 62 +++++++++++++++++++++++
 arch/arm/dts/armada-3720-ddr3-db-v2-B.dts | 55 ++++++++++++++++++++
 arch/arm/dts/armada-3720-ddr3-db-v2-C.dts | 62 +++++++++++++++++++++++
 arch/arm/dts/armada-3720-ddr4-db-v1-A.dts | 47 +++++++++++++++++
 arch/arm/dts/armada-3720-ddr4-db-v1-B.dts | 56 ++++++++++++++++++++
 arch/arm/dts/armada-3720-ddr4-db-v3-A.dts | 33 ++++++++++++
 arch/arm/dts/armada-3720-ddr4-db-v3-B.dts | 56 ++++++++++++++++++++
 10 files changed, 478 insertions(+)
 create mode 100644 arch/arm/dts/armada-3720-ddr3-db-v1-A.dts
 create mode 100644 arch/arm/dts/armada-3720-ddr3-db-v1-B.dts
 create mode 100644 arch/arm/dts/armada-3720-ddr3-db-v1-C.dts
 create mode 100644 arch/arm/dts/armada-3720-ddr3-db-v2-B.dts
 create mode 100644 arch/arm/dts/armada-3720-ddr3-db-v2-C.dts
 create mode 100644 arch/arm/dts/armada-3720-ddr4-db-v1-A.dts
 create mode 100644 arch/arm/dts/armada-3720-ddr4-db-v1-B.dts
 create mode 100644 arch/arm/dts/armada-3720-ddr4-db-v3-A.dts
 create mode 100644 arch/arm/dts/armada-3720-ddr4-db-v3-B.dts

diff --git a/arch/arm/dts/Makefile b/arch/arm/dts/Makefile
index ee19d2394b..a686340e4f 100644
--- a/arch/arm/dts/Makefile
+++ b/arch/arm/dts/Makefile
@@ -171,6 +171,15 @@ dtb-$(CONFIG_TEGRA) += tegra20-harmony.dtb \
 
 dtb-$(CONFIG_ARCH_MVEBU) +=			\
 	armada-3720-db.dtb			\
+	armada-3720-ddr3-db-v2-B.dtb		\
+	armada-3720-ddr3-db-v2-C.dtb		\
+	armada-3720-ddr3-db-v1-A.dtb		\
+	armada-3720-ddr3-db-v1-B.dtb		\
+	armada-3720-ddr3-db-v1-C.dtb		\
+	armada-3720-ddr4-db-v3-A.dtb		\
+	armada-3720-ddr4-db-v3-B.dtb		\
+	armada-3720-ddr4-db-v1-A.dtb		\
+	armada-3720-ddr4-db-v1-B.dtb		\
 	armada-3720-espressobin.dtb		\
 	armada-3720-turris-mox.dtb		\
 	armada-3720-uDPU.dtb			\
diff --git a/arch/arm/dts/armada-3720-ddr3-db-v1-A.dts b/arch/arm/dts/armada-3720-ddr3-db-v1-A.dts
new file mode 100644
index 0000000000..4fe02d0acc
--- /dev/null
+++ b/arch/arm/dts/armada-3720-ddr3-db-v1-A.dts
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: GPL-2.0+
+/*
+ * Copyright (C) 2018 Marvell International Ltd.
+ */
+
+#include "armada-3720-db.dts"
+
+/* detail board setup:
+ * Pcie Jumper slot	:	DB-88F3720-PCIe-mPCIe-Jumper
+ * SGMII module slot	:	DB-88F3720-SerDes-Jumper
+ * RGMII module slot	:	DB-88F3720-PHY module
+ * eMMC slot		:	DB-88F3720-eMMC_NAND module
+ * SDIO slot		:	DB-88F3720-SDcard module
+ *
+ * PHY0 (USB3/SGMII1)	:	USB3
+ * PHY1 (PCIe/SGMII0)	:	PCIe (WiFi via mini_PCIe module)
+ * PHY2 (SATA)		:	SATA
+ * RGMII-0		:	Ethernet via PHY (1 Lane)
+ * USB2 Port 0		:	USB2 as part USB3
+ * USB2 Port 1		:	USB2 Host
+ * eMMC*		:	eMMC NAND,
+				connected to SD / eMMC Host 1 in North Bridge
+ * SDIO*		:	SD card,
+				connected to SDIO Host 0 in South Bridge
+ */
+
+/ {
+	model = "Marvell Armada 3720 Development Board DB-88F3720-DDR3-V1";
+	compatible = "marvell,armada-3720-db-v1",
+		     "marvell,armada3720",
+		     "marvell,armada3710";
+
+	soc {
+		internal-regs {
+			comphy: comphy@18300 {
+				phy2 {
+					phy-invert =
+						<COMPHY_POLARITY_TXD_INVERT>;
+				};
+			};
+		};
+	};
+};
diff --git a/arch/arm/dts/armada-3720-ddr3-db-v1-B.dts b/arch/arm/dts/armada-3720-ddr3-db-v1-B.dts
new file mode 100644
index 0000000000..c8e043ccd9
--- /dev/null
+++ b/arch/arm/dts/armada-3720-ddr3-db-v1-B.dts
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: GPL-2.0+
+/*
+ * Copyright (C) 2018 Marvell International Ltd.
+ */
+
+#include "armada-3720-ddr3-db-v1-A.dts"
+
+/* detail board setup:
+ * Pcie Jumper slot	:	DB-88F3720-PCIe-mPCIe-Jumper
+ * SGMII module slot	:	DB-88F3720-SerDes-Jumper
+ * RGMII module slot	:	DB-88F3720-PHY module
+ * eMMC slot		:	DB-88F3720-eMMC_NAND module
+ * SDIO slot		:	DB-88F3720-SDcard module
+ *
+ * PHY0 (USB3/SGMII1)	:	SGMII1_Ethernet via PHY (1 Lane)
+ * PHY1 (PCIe/SGMII0)	:	PCIe (WiFi via mini_PCIe module)
+ * PHY2 (SATA)		:	SATA
+ * RGMII		:	Ethernet via PHY (1 Lane)
+ * USB2 Port 0		:	USB2 only
+ * USB2 Port 1		:	USB2 Host
+ * eMMC*		:	eMMC NAND,
+				connected to SD / eMMC Host 1 in North Bridge
+ * SDIO*		:	SD card,
+				connected to SDIO Host 0 in South Bridge
+ */
+
+/ {
+	model = "Marvell Armada 3720 Development Board DB-88F3720-DDR3-V1";
+	compatible = "marvell,armada-3720-db-v1",
+		     "marvell,armada3720",
+		     "marvell,armada3710";
+
+	soc {
+		internal-regs {
+			comphy: comphy@18300 {
+				phy0 {
+					phy-type = <COMPHY_TYPE_SGMII1>;
+					phy-speed = <COMPHY_SPEED_1_25G>;
+				};
+			};
+			mdio@32004 {
+				#address-cells = <1>;
+				#size-cells = <0>;
+				ethphy1: ethernet-phy@1 {
+					reg = <1>;
+				};
+			};
+			eth1: neta@40000 {
+				status = "okay";
+				phy-mode = "sgmii";
+				phy = <&ethphy1>;
+			};
+		};
+	};
+};
diff --git a/arch/arm/dts/armada-3720-ddr3-db-v1-C.dts b/arch/arm/dts/armada-3720-ddr3-db-v1-C.dts
new file mode 100644
index 0000000000..c078c0702c
--- /dev/null
+++ b/arch/arm/dts/armada-3720-ddr3-db-v1-C.dts
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: GPL-2.0+
+/*
+ * Copyright (C) 2018 Marvell International Ltd.
+ */
+
+#include "armada-3720-ddr3-db-v1-A.dts"
+
+/* detail board setup:
+ * Pcie Jumper slot	:	DB-88F3720-PCIe-mPCIe-Jumper
+ * SGMII module slot	:	DB-88F3720-SerDes-Jumper
+ * RGMII module slot	:	DB-88F3720-PHY module
+ * eMMC slot		:	DB-88F3720-eMMC_NAND module
+ * SDIO slot		:	DB-88F3720-SDcard module
+ *
+ * PHY0 (USB3/SGMII1)	:	SGMII1_Ethernet via SFP2.5G (1 Lane)
+ * PHY1 (PCIe/SGMII0)	:	PCIe (WiFi via mini_PCIe module)
+ * PHY2 (SATA)		:	SATA
+ * RGMII		:	Ethernet via PHY (1 Lane)
+ * USB2 Port 0		:	USB2 only
+ * USB2 Port 1		:	USB2 Host
+ * eMMC*		:	eMMC NAND,
+				connected to SD / eMMC Host 1 in North Bridge
+ * SDIO*		:	SD card,
+				connected to SDIO Host 0 in South Bridge
+ */
+
+/ {
+	model = "Marvell Armada 3720 Development Board DB-88F3720-DDR3-V1";
+	compatible = "marvell,armada-3720-db-v1",
+		     "marvell,armada3720",
+		     "marvell,armada3710";
+
+	soc {
+		internal-regs {
+			comphy: comphy@18300 {
+				phy0 {
+					phy-type = <COMPHY_TYPE_SGMII1>;
+					phy-speed = <COMPHY_SPEED_3_125G>;
+				};
+			};
+			eth1: neta@40000 {
+				status = "okay";
+				phy-mode = "sgmii";
+				fixed-link {
+					/* Here "speed" is set to 1000, GBE MAC
+					 * is running in 1G mode, but the
+					 * actuall PHY speed may be 1 Gbps or
+					 * 2.5 Gbps, it's up to the
+					 * corresponding SERDES speed in comphy
+					 * node.
+					 * If SERDES speed is set to 3.125G, it
+					 * implies sgmii 2.5 Gbps;
+					 * if SERDES speed is set to 1.25G, it
+					 * implies sgmii 1 Gbps.
+					 */
+					speed = <1000>;
+					full-duplex;
+				};
+			};
+		};
+	};
+};
diff --git a/arch/arm/dts/armada-3720-ddr3-db-v2-B.dts b/arch/arm/dts/armada-3720-ddr3-db-v2-B.dts
new file mode 100644
index 0000000000..5ab4525501
--- /dev/null
+++ b/arch/arm/dts/armada-3720-ddr3-db-v2-B.dts
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: GPL-2.0+
+/*
+ * Copyright (C) 2018 Marvell International Ltd.
+ */
+
+#include "armada-3720-db.dts"
+
+/* detail board setup:
+ * Pcie Jumper slot	:	DB-88F3720-PCIe-mPCIe-Jumper
+ * SGMII module slot	:	DB-88F3720-SerDes-Jumper
+ * RGMII module slot	:	DB-88F3720-PHY module
+ * eMMC slot		:	DB-88F3720-eMMC_NAND module
+ * SDIO slot		:	DB-88F3720-SDcard module
+ *
+ * PHY0 (USB3/SGMII1)	:	SGMII1_Ethernet via PHY (1 Lane)
+ * PHY1 (PCIe/SGMII0)	:	PCIe (WiFi via mini_PCIe module)
+ * PHY2 (SATA)		:	SATA
+ * RGMII		:	Ethernet via PHY (1 Lane)
+ * USB2 Port 0		:	USB2 only
+ * USB2 Port 1		:	USB2 Host
+ * eMMC*		:	eMMC NAND,
+				connected to SD / eMMC Host 1 in North Bridge
+ * SDIO*		:	SD card,
+				connected to SDIO Host 0 in South Bridge
+ */
+
+/ {
+	model = "Marvell Armada 3720 Development Board DB-88F3720-DDR3-V2";
+	compatible = "marvell,armada-3720-db-v2",
+		     "marvell,armada3720",
+		     "marvell,armada3710";
+
+	soc {
+		internal-regs {
+			comphy: comphy@18300 {
+				phy0 {
+					phy-type = <COMPHY_TYPE_SGMII1>;
+					phy-speed = <COMPHY_SPEED_1_25G>;
+				};
+			};
+			mdio@32004 {
+				#address-cells = <1>;
+				#size-cells = <0>;
+				ethphy1: ethernet-phy@1 {
+					reg = <1>;
+				};
+			};
+			eth1: neta@40000 {
+				status = "okay";
+				phy-mode = "sgmii";
+				phy = <&ethphy1>;
+			};
+		};
+	};
+};
diff --git a/arch/arm/dts/armada-3720-ddr3-db-v2-C.dts b/arch/arm/dts/armada-3720-ddr3-db-v2-C.dts
new file mode 100644
index 0000000000..283cabcd25
--- /dev/null
+++ b/arch/arm/dts/armada-3720-ddr3-db-v2-C.dts
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: GPL-2.0+
+/*
+ * Copyright (C) 2018 Marvell International Ltd.
+ */
+
+#include "armada-3720-db.dts"
+
+/* detail board setup:
+ * Pcie Jumper slot	:	DB-88F3720-PCIe-mPCIe-Jumper
+ * SGMII module slot	:	DB-88F3720-SerDes-Jumper
+ * RGMII module slot	:	DB-88F3720-PHY module
+ * eMMC slot		:	DB-88F3720-eMMC_NAND module
+ * SDIO slot		:	DB-88F3720-SDcard module
+ *
+ * PHY0 (USB3/SGMII1)	:	SGMII1_Ethernet via SFP2.5G (1 Lane)
+ * PHY1 (PCIe/SGMII0)	:	PCIe (WiFi via mini_PCIe module)
+ * PHY2 (SATA)		:	SATA
+ * RGMII		:	Ethernet via PHY (1 Lane)
+ * USB2 Port 0		:	USB2 only
+ * USB2 Port 1		:	USB2 Host
+ * eMMC*		:	eMMC NAND,
+				connected to SD / eMMC Host 1 in North Bridge
+ * SDIO*		:	SD card,
+				connected to SDIO Host 0 in South Bridge
+*/
+
+/ {
+	model = "Marvell Armada 3720 Development Board DB-88F3720-DDR3-V2";
+	compatible = "marvell,armada-3720-db-v2",
+		     "marvell,armada3720",
+		     "marvell,armada3710";
+
+	soc {
+		internal-regs {
+			comphy: comphy@18300 {
+				phy0 {
+					phy-type = <COMPHY_TYPE_SGMII1>;
+					phy-speed = <COMPHY_SPEED_3_125G>;
+				};
+			};
+			eth1: neta@40000 {
+				status = "okay";
+				phy-mode = "sgmii";
+				fixed-link {
+					/* Here "speed" is set to 1000, GBE MAC
+					 * is running in 1G mode, but the
+					 * actuall PHY speed may be 1 Gbps or
+					 * 2.5 Gbps, it's up to the
+					 * corresponding SERDES speed in comphy
+					 * node.
+					 * If SERDES speed is set to 3.125G, it
+					 * implies sgmii 2.5 Gbps;
+					 * if SERDES speed is set to 1.25G, it
+					 * implies sgmii 1 Gbps.
+					 */
+					speed = <1000>;
+					full-duplex;
+				};
+			};
+		};
+	};
+};
diff --git a/arch/arm/dts/armada-3720-ddr4-db-v1-A.dts b/arch/arm/dts/armada-3720-ddr4-db-v1-A.dts
new file mode 100644
index 0000000000..76545695cd
--- /dev/null
+++ b/arch/arm/dts/armada-3720-ddr4-db-v1-A.dts
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: GPL-2.0+
+/*
+ * Copyright (C) 2018 Marvell International Ltd.
+ */
+
+#include "armada-3720-ddr3-db-v1-A.dts"
+
+/* detail board setup:
+ * Pcie Jumper slot	:	DB-88F3720-PCIe-mPCIe-Jumper
+ * SGMII module slot	:	DB-88F3720-SerDes-Jumper
+ * RGMII module slot	:	DB-88F3720-PHY module
+ * eMMC slot		:	DB-88F3720-eMMC_NAND module
+ * SDIO slot		:	DB-88F3720-SDcard module
+ *
+ * PHY0 (USB3/SGMII1)	:	USB3
+ * PHY1 (PCIe/SGMII0)	:	PCIe (WiFi via mini_PCIe module)
+ * PHY2 (SATA)		:	SATA
+ * RGMII-0		:	Ethernet via PHY (1 Lane)
+ * USB2 Port 0		:	USB2 as part USB3
+ * USB2 Port 1		:	USB2 Host
+ * eMMC*		:	eMMC NAND,
+				connected to SD / eMMC Host 1 in North Bridge
+ * SDIO*		:	SD card,
+				connected to SDIO Host 0 in South Bridge
+ */
+
+/ {
+	model = "Marvell Armada 3720 Development Board DB-88F3720-DDR4-V1";
+	compatible = "marvell,armada-3720-db-v2",
+		     "marvell,armada3720",
+		     "marvell,armada3710";
+
+	soc {
+		internal-regs {
+			comphy: comphy@18300 {
+				phy2 {
+					phy-invert =
+						<COMPHY_POLARITY_NO_INVERT>;
+				};
+			};
+		};
+	};
+
+	vcc_sd_reg0: regulator@0 {
+		gpios = <&gpiosb 22 GPIO_ACTIVE_HIGH>;
+	};
+};
diff --git a/arch/arm/dts/armada-3720-ddr4-db-v1-B.dts b/arch/arm/dts/armada-3720-ddr4-db-v1-B.dts
new file mode 100644
index 0000000000..dccacdca2f
--- /dev/null
+++ b/arch/arm/dts/armada-3720-ddr4-db-v1-B.dts
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: GPL-2.0+
+/*
+ * Copyright (C) 2018 Marvell International Ltd.
+ */
+
+#include "armada-3720-ddr4-db-v1-A.dts"
+
+/* detail board setup:
+ * Pcie Jumper slot	:	DB-88F3720-PCIe-mPCIe-Jumper
+ * SGMII module slot	:	DB-88F3720-SerDes-Jumper
+ * RGMII module slot	:	DB-88F3720-PHY module
+ * eMMC slot		:	DB-88F3720-eMMC_NAND module
+ * SDIO slot		:	DB-88F3720-SDcard module
+ *
+ * PHY0 (USB3/SGMII1)	:	SGMII1
+ * PHY1 (PCIe/SGMII0)	:	PCIe (WiFi via mini_PCIe module)
+ * PHY2 (SATA)		:	SATA
+ * RGMII-0		:	Ethernet via PHY (1 Lane)
+ * USB2 Port 0		:	USB2 as part USB3
+ * USB2 Port 1		:	USB2 Host
+ * eMMC*		:	eMMC NAND,
+				connected to SD / eMMC Host 1 in North Bridge
+ * SDIO*		:	SD card,
+				connected to SDIO Host 0 in South Bridge
+ */
+
+/ {
+	model = "Marvell Armada 3720 Development Board DB-88F3720-DDR4-V1";
+	compatible = "marvell,armada-3720-db-v2",
+		     "marvell,armada3720",
+		     "marvell,armada3710";
+
+	soc {
+		internal-regs {
+			comphy: comphy@18300 {
+				phy0 {
+					phy-type = <COMPHY_TYPE_SGMII1>;
+					phy-speed = <COMPHY_SPEED_1_25G>;
+				};
+			};
+			mdio@32004 {
+				#address-cells = <1>;
+				#size-cells = <0>;
+				ethphy1: ethernet-phy@1 {
+					reg = <1>;
+				};
+			};
+			eth1: neta@40000 {
+				status = "okay";
+				phy-mode = "sgmii";
+				phy = <&ethphy1>;
+			};
+		};
+	};
+
+};
diff --git a/arch/arm/dts/armada-3720-ddr4-db-v3-A.dts b/arch/arm/dts/armada-3720-ddr4-db-v3-A.dts
new file mode 100644
index 0000000000..a956ec6cc4
--- /dev/null
+++ b/arch/arm/dts/armada-3720-ddr4-db-v3-A.dts
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: GPL-2.0+
+/*
+ * Copyright (C) 2018 Marvell International Ltd.
+ */
+
+#include "armada-3720-db.dts"
+
+/* detail board setup:
+ * Pcie Jumper slot	:	DB-88F3720-PCIe-mPCIe-Jumper
+ * SGMII module slot	:	DB-88F3720-SerDes-Jumper
+ * RGMII module slot	:	DB-88F3720-PHY module
+ * eMMC slot		:	DB-88F3720-eMMC_NAND module
+ * SDIO slot		:	DB-88F3720-SDcard module
+ *
+ * PHY0 (USB3/SGMII1)	:	USB3
+ * PHY1 (PCIe/SGMII0)	:	PCIe (WiFi via mini_PCIe module)
+ * PHY2 (SATA)		:	SATA
+ * RGMII-0		:	Ethernet via PHY (1 Lane)
+ * USB2 Port 0		:	USB2 as part USB3
+ * USB2 Port 1		:	USB2 Host
+ * eMMC*		:	eMMC NAND,
+				connected to SD / eMMC Host 1 in North Bridge
+ * SDIO*		:	SD card,
+				connected to SDIO Host 0 in South Bridge
+ */
+
+/ {
+	model = "Marvell Armada 3720 Development Board DB-88F3720-DDR4-V3";
+	compatible = "marvell,armada-3720-db-v3",
+		     "marvell,armada3720",
+		     "marvell,armada3710";
+
+};
diff --git a/arch/arm/dts/armada-3720-ddr4-db-v3-B.dts b/arch/arm/dts/armada-3720-ddr4-db-v3-B.dts
new file mode 100644
index 0000000000..882151a96a
--- /dev/null
+++ b/arch/arm/dts/armada-3720-ddr4-db-v3-B.dts
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: GPL-2.0+
+/*
+ * Copyright (C) 2018 Marvell International Ltd.
+ */
+
+#include "armada-3720-ddr4-db-v3-A.dts"
+
+/* detail board setup:
+ * Pcie Jumper slot	:	DB-88F3720-PCIe-mPCIe-Jumper
+ * SGMII module slot	:	DB-88F3720-SerDes-Jumper
+ * RGMII module slot	:	DB-88F3720-PHY module
+ * eMMC slot		:	DB-88F3720-eMMC_NAND module
+ * SDIO slot		:	DB-88F3720-SDcard module
+ *
+ * PHY0 (USB3/SGMII1)	:	SGMII1
+ * PHY1 (PCIe/SGMII0)	:	PCIe (WiFi via mini_PCIe module)
+ * PHY2 (SATA)		:	SATA
+ * RGMII-0		:	Ethernet via PHY (1 Lane)
+ * USB2 Port 0		:	USB2 as part USB3
+ * USB2 Port 1		:	USB2 Host
+ * eMMC*		:	eMMC NAND,
+				connected to SD / eMMC Host 1 in North Bridge
+ * SDIO*		:	SD card,
+				connected to SDIO Host 0 in South Bridge
+ */
+
+/ {
+	model = "Marvell Armada 3720 Development Board DB-88F3720-DDR4-V3";
+	compatible = "marvell,armada-3720-db-v3",
+		     "marvell,armada3720",
+		     "marvell,armada3710";
+
+	soc {
+		internal-regs {
+			comphy: comphy@18300 {
+				phy0 {
+					phy-type = <COMPHY_TYPE_SGMII1>;
+					phy-speed = <COMPHY_SPEED_1_25G>;
+				};
+			};
+			mdio@32004 {
+				#address-cells = <1>;
+				#size-cells = <0>;
+				ethphy1: ethernet-phy@1 {
+					reg = <1>;
+				};
+			};
+			eth1: neta@40000 {
+				status = "okay";
+				phy-mode = "sgmii";
+				phy = <&ethphy1>;
+			};
+		};
+	};
+
+};
-- 
2.29.0

